sockeye/model.py

```diff
@@ -68,6 +74,10 @@ def __init__(self,
         self.config_loss = config_loss
         self.lexical_bias = lexical_bias
         self.learn_lexical_bias = learn_lexical_bias
+        self.weight_tying = weight_tying
+        if weight_tying and weight_tying_type is None:
```

Comment:
only if someone specifically passes None, which is actually what I'm doing from train.py. The reason is that it looks slightly confusing when the model/config says:
```
weight_tying: False
weight_tying_type: trg_softmax
```
When setting this to None in case of `weight_tying=False` we get:
```
weight_tying: False
weight_tying_type: null
```
Anyway, just a minor detail. So it can happen.